src/components/ListView.js

```diff
@@ -53,22 +83,34 @@ const ListView = ({ shoppingList, loading, error }) => {
           </form>
 
           <ul className="list">
-            {shoppingList &&
-              shoppingList[0] &&
-              shoppingList[0].items.map((item) => {
-                const searchResult = item.name.includes(
-                  value.toLowerCase().trim(),
-                );
-                return (
-                  searchResult && (
-                    <ListItem
-                      key={item.id}
-                      item={item}
-                      shoppingList={shoppingList}
-                    />
-                  )
-                );
-              })}
+            {sortedList.map((item) => {
+              const searchResult = item.name.includes(
+                value.toLowerCase().trim(),
+              );
+              //pass parameters to checkIndex function
+              const lastPurchase = item.LastPurchasedDate;
+              const nextPurchase = item.daysLeftForNextPurchase;
+              const timesPurchased = item.numberOfPurchases;
+              //check urgency index of item
+              checkIndex(
+                lastPurchase,
+                nextPurchase,
+                timesPurchased,
+                purchaseIndex,
+              );
+
+              return (
+                searchResult && (
+                  <ListItem
+                    key={item.id}
+                    item={item}
+                    //index prop assigns className and aria-label to item
+                    index={purchaseIndex}
```

Comment:
Since we are giving the whole list item background colour. Can't we add the className directly here instead of passing it as a prop in the listItem? Unless we want the colour to only be on the checkbox?

Reply:
Passing it as a prop in the List Item adds the background colour to the whole item, not only to the checkbox. 

If you view the app using the preview link, you can view/inspect that. Thank you!

Reply:
Yeah I get, so this is what I was suggesting
![image](https://user-images.githubusercontent.com/37863089/118285647-fa927980-b4d1-11eb-99b3-4c7c1a5950c4.png)

Reply:
Tried it, and it doesn't work.

Reply:
oooh okay.